tests/mir-opt/const_prop/overwrite_with_const_with_params.rs

```diff
@@ -0,0 +1,23 @@
+// unit-test: ConstProp
+// compile-flags: -O
+// skip-filecheck
```

Comment:
Could you add the filecheck annotations? See other files for example.

Reply:
done